kubernetes/kuma-demo-aio.yaml

```diff
@@ -259,5 +271,5 @@ spec:
       containers:
       - name: kuma-fe
         image: kvn0218/kuma-demo-fe:latest
```

Comment:
Maybe not as a part of this PR, but shouldn't we use image placed under the official kuma accounts? I'm not even sure what is this user `kvn0218`.

Reply:
Opened an issue about it: https://github.com/kumahq/kuma-demo/issues/160